### PR TITLE
created rounded image and images container components

### PR DESCRIPTION
**Pull request by zidano88**
will modify data to be a Json and use map method

### CODE REVIEW

**Comment by zidano88** [comment]
Updated the file and used Json and map method

File: src/components/EvComponents/RoundedImages/RoundedImagesPanel.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        name: "Speaker 12",
+        title: "Head Manager 12"
+    },
+];
+
+const RoundedImagesPanel = () => {
+    return (
+        <Grid container columns={{ xs: 1, sm: 3, md: 6 }} sx={{ padding: "35px" }}>
+
+            {Array.from(imagesData).map((_, index) => (
```
Comment (7ganam) [inline src/components/EvComponents/RoundedImages/RoundedImagesPanel.jsx]: this is not the best way to use array maps.. you don't need to use index ...

File: src/components/EvComponents/RoundedImages/RoundedImagesPanel.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                <Grid item xs={12} sm={1} md={1} key={index}>
+                    <RoundedImage Src={imagesData[index].imageSrc} Name={imagesData[index].name} Title={imagesData[index].title} />
+                </Grid>
+            ))}
+
+            {/* <Grid item xs={12} sm={1} md={1}>
```
Comment (7ganam) [inline src/components/EvComponents/RoundedImages/RoundedImagesPanel.jsx]: remove commments

File: src/components/EvComponents/RoundedImages/RoundedImage.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { Box } from "@mui/system";
+import React from "react";
+import Image from "components/BazarImage";
+import { Paragraph } from "components/EvComponents/Typography";
+import { H1, H2, H3, H4 } from "components/EvComponents/Typography";
+// import { BlockTitle } from "components/EvComponents/StyledTypography";
```
Comment (7ganam) [inline src/components/EvComponents/RoundedImages/RoundedImage.jsx]: remove comment

File: pages/gallery.jsx [hunk 4]
```diff
@@ -6,6 +6,8 @@ import TextSection from "components/EvSections/TextSection";
 import EvLayout from "components/layouts/EvLayout";
 
 import api from "utils/api/grocery3-shop";
+import RoundedImagesPanel from "components/EvComponents/RoundedImages/RoundedImagesPanel";
```
Comment (7ganam) [inline pages/gallery.jsx]: remove the RoundedImagesPanel from the EvComponents to EvSections and change its name to RoundedImagesSection

**Review by 7ganam — CHANGES_REQUESTED** [review]

File: src/components/EvComponents/RoundedImages/RoundedImagesPanel.jsx (hunk 2 above)
Comment (zidano88) [reply src/components/EvComponents/RoundedImages/RoundedImagesPanel.jsx]: done

File: src/components/EvComponents/RoundedImages/RoundedImagesPanel.jsx (hunk 1 above)
Comment (zidano88) [reply src/components/EvComponents/RoundedImages/RoundedImagesPanel.jsx]: modified and removed index

File: src/components/EvComponents/RoundedImages/RoundedImage.jsx (hunk 3 above)
Comment (zidano88) [reply src/components/EvComponents/RoundedImages/RoundedImage.jsx]: removed

File: pages/gallery.jsx (hunk 4 above)
Comment (zidano88) [reply pages/gallery.jsx]: changed position and name

**Review by 7ganam — APPROVED** [review]

**Pull request merged**